recipes/netgen/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+    name: netgen
+    version: 6.2
+
+source:
+    fn: netgen.tar.gz
+    url: https://github.com/NGSolve/netgen/archive/a1d1df91b7912a7acb9864301cb108b2e01b4882.tar.gz
```

Comment:
Would you mind using the standard jinja2 variables (at least for the url)?

Reply:
thanks, done.